usr/src/uts/common/fs/zfs/arc.c

```diff
@@ -284,10 +285,11 @@ boolean_t arc_watch = B_FALSE;
 int arc_procfd;
 #endif
 
-static kmutex_t		arc_reclaim_lock;
-static kcondvar_t	arc_reclaim_thread_cv;
-static boolean_t	arc_reclaim_thread_exit;
-static kcondvar_t	arc_reclaim_waiters_cv;
+static zthr_t		*arc_reap_zthr;
```

Comment:
Use/modify text from the bug report to state here what these two zthreads do.

Reply:
Good idea.